intermediate_source/nvfuser_intro_tutorial.py

```diff
@@ -0,0 +1,647 @@
+# -*- coding: utf-8 -*-
+"""
+Getting Started - Accelerate Your Scripts with nvFuser
+****************************
+
+**Authors**: `Christian Sarofeen <https://github.com/csarofeen>`_
+`Kevin Stephano <https://github.com/kevinstephano>`_
+`Piotr Bialecki <https://github.com/ptrblck>`_
+`Jie Jiang <https://github.com/jjsjann123>`_
+`Masaki Kozuki <https://github.com/crcrpar>`_
+`Neal Vaidya`
+
+
+Introduction
+------------
+
+This tutorial will demonstrate how you can accelerate your networks
+with nvFuser. nvFuser is a Deep Learning Compiler that just-in-time
+compiles fast and flexible GPU specific code to reliably accelerate
+users' networks automatically, providing speedups for deep learning
+networks running on Volta and later CUDA accelerators by generating
+fast custom “fusion” kernels at runtime. nvFuser is specifically
+designed to meet the unique requirements of the PyTorch community,
+and it supports diverse network architectures and programs with
+dynamic inputs of varying shapes and strides. If you’re unfamiliar
+with nvFuser please review [this blog post, INSERT BLOG POST LINK].
+
+Importing Packages and Selecting a Device
+-----------------------------------------
+In order to run this tutorial and see the benefits of using nvfuser,
+you would need to install the `1.12.0` PyTorch release as well as
+`functorch` `0.2` or newer version of them.
+Additionally, a GPU is required.
+"""
+
+from functorch.compile import memory_efficient_fusion
+from torch.utils import collect_env
+import torch.nn.functional as F
+import torch
+from copy import deepcopy
+from typing import List
+import time
+import functools
+import random
+random.seed(42)
+
+
+print(collect_env.get_pretty_env_info())
+
+######################################################################
+# The Transformer Block
+# ---------------------
+# The network topology we’re going to focus on is the Transformer
+# Block for networks like BERT. As of writing this blog post nvFuser
+# provides acceleration of pointwise, reduction, and normalization
+# operations. These simple operations are the backbone of large
+# networks, so improving the speed of these operations can improve
+# overall network training speed. Future releases nvFuser will
```

Comment:
Does "Future releases nvFuser will" need some preposition (a) before "Future" or (b) between "releases" and "nvFuser"?

Reply:
I go with b):
> Future releases *of* nvFuser...